hw2/02-reverse.js

```diff
@@ -1,3 +1,30 @@
-/** Exercise 02 - Reverse **/
+/** Exercise 02 - Reverse * */
+const input = document.querySelector('#input')
+const reverse = document.querySelector('#reverse')
+const result = document.querySelector('#result')
 
-// Add your code here
+const setError = (msg) => {
+    result.style.color = '#990505'
+    result.textContent = `Error: ${msg}`
+}
+
+const setResult = (original, reversed) => {
+    result.style.color = '#045502'
+    result.textContent = `${original} --> ${reversed}`
+}
```

Comment:
this works great. another way you could set the color is through adding/changing a bootstrap native class such as "valid-input" and "invalid-input" to the element.